conda.recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: param
-  version: {{environ['GIT_DESCRIBE_TAG'].lstrip('v').split('.dev')[0]}}.dev{{GIT_BUILD_STR }}
+  version: {{environ['GIT_DESCRIBE_TAG'].lstrip('v')}}.post{{GIT_DESCRIBE_NUMBER}}.dev+{{GIT_DESCRIBE_HASH}}
```

Comment:
Hmm.  If it's .yml, I don't know how to do the arithmetic necessary to use the "predicted next release form"; how will it increment the result of GIT_DESCRIBE_TAG?

Reply:
If setup.py had the correct version, I would instead use that. Something like:

```
{% set setup_data = load_setup_py_data() %}

package:
  name: nbsite
  version: {{ setup_data.get('version') }}
```

The version in setup.py can be computed by python. (So in projects where I've used a versioning tool to ensure the version's always up to date from git, there's no need to do the "manual version construction" I've done in this PR.)